### ISSUE TITLE
Nav bar improvements

### ISSUE THREAD

**Issue by RLC-DCPPC** [nih-cfde/cfde-deriva#54]
For the demo web page, a few workarounds were added for the navbar since directly including it caused some styling collisions and other functionality to not work. The problem occurred because of a dependency on 2 different versions of `bootstrap.css` (`3.3.7` and `4.7.0`). Both were loaded into the environment side by side and that caused the clash in styles in navbar and some odd functionality with the modal's behavior since it's using a CSS solution to show/hide itself.

As a temporary solution, the navbar was placed inside of an `<iframe>` for the landing page and dashboard. This was done to scope the dependencies for the navbar so the version of bootstrap for the landing page and navbar could be different. This caused a few other issues, notably the navbar takes longer to load and the dropdowns in the navbar get clipped (so they are currently turned off).

**Comment by RLC-DCPPC** [comment]
Hi @jrchudy . @victor73 told me you were working on this, but I doubt if I described it well. Can you two fix the description and give a status on progress. Sounds like something we'd like to see in the next release of the UI.

**Comment by jrchudy** [comment]
@RLC-DCPPC I made edits to the description above to more accurately describe the issue that occurred during development for the demo which prompted the use of the iframe. I detailed the issue from the perspective of chaise and what the limitations from the deriva team are.

Chaise is using `bootstrap.css` `3.3.7`, which is required for the version of ui.bootstrap used (an angularJS library). Chaise is using AngularJS `1.6.9` which locks the versions for a lot of the dependencies.

**Comment by RLC-DCPPC** [comment]
would like to get away from reliance on i-frame

currently, drop-downs don't work in navbar (in particular, login)
redirection is forcing new tabs

consider solutions to this version collision. edplore our dependence on a particular version of bootstrap

**Comment by jrchudy** [comment]
I have plugged away a bit at this today to figure out exactly what styles changed in both places and came up with some fixes that can be found here:
https://app.nih-cfde.org/sample-navbar.html

1. I was able to remove the iframe and fix the navbar styles by adding more rules to `/var/www/html/css/custom.css` to remove or modify ones that bootstrap was changing.

2. I added another specific style into that document targeting `#form-controls .container` to fix the width issue that the inputs had.

3. I removed the `fade` and `show` classes that were defined on the modal (`#export-modal`).
  - with these 2 classes on the modal, it would show by default and by "faded" or invisible
  - the functionality still works without these 2 classes defined

4. In `/var/www/html/css/style.css`, I removed the rule targeting `header` that was adding a padding and background color.
